clang/test/SemaSYCL/sycl-dllimport-dllexport.cpp

```diff
@@ -0,0 +1,48 @@
+// RUN: %clang_cc1 -triple spir64-unknown-linux-sycldevice -fsycl-is-device -fms-extensions -DTEST %s -o /dev/null 2>&1 | FileCheck %s --check-prefixes CHECK
+
+// RUN: %clang_cc1 -triple spir64-unknown-windows-sycldevice -fms-extensions -DTEST %s -o /dev/null 2>&1 | FileCheck %s --check-prefixes CHECK
```

Comment:
Interesting....
What is expected from the clang by setting `sycldevice` environment triple component without setting `-fsycl-is-device` flag?

Reply:
Same a negative testing...  Just get combination to checked waning